src/ark/segmentation/ez_seg/composites.py

```diff
@@ -73,28 +76,34 @@ def composite_builder(
                 fov_data, composite_array, images_to_subtract, image_type, composite_method
             )
 
-        # Create the fov dir within the composite dir
-        composite_fov_dir = composite_directory / fov
-        composite_fov_dir.mkdir(parents=True, exist_ok=True)
+        if composite_directory:
```

Comment:
Above on lines 64-66, this creates the composite directory if it exists, do you want to remove lines 64-66? (duplicate code more or less) since the directories get created here anyways.